tests/integration/framework/testEnvManager.go

```diff
@@ -15,48 +15,55 @@
 package framework
 
 import (
+	"flag"
 	"fmt"
 	"log"
 	"time"
 
-	tu "istio.io/istio/tests/util"
+	u "istio.io/istio/tests/util"
+)
+
+var (
+	skipCleanup = flag.Bool("skip_cleanup", false, "Debug, skip clean up")
 )
 
 // runnable is used for Testing purposes.
 type runnable interface {
 	Run() int
 }
 
-// IstioTestFramework is core test framework struct
-type IstioTestFramework struct {
-	TestEnv    TestEnv
-	TestID     string
-	Components []Component
+// TestEnvManager is core test framework struct
+type TestEnvManager struct {
```

Comment:
Personally, I prefer "manager" or may be some other words since it takes care of the whole cycle of a environment: start, stop, alive check. 

And I plan to add a parallel health monitor for the environment such that when tests failed in the middle we can see if the component crash is the reason.